.tool-versions

```diff
@@ -1 +0,0 @@
-elixir 1.4.2
```

Comment:
I think I rage deleted this and didn't mean to push up that change. Do we want a `.tool-versions` file in a library? I can put it back if anyone has any objections to its removal.